tests/test_project/tests/test_admin.py

```diff
@@ -95,3 +96,17 @@ def test_additional_buttons(self):
         response = self.client.get(url)
         self.assertContains(response, 'Preview book')
         self.assertContains(response, 'Download book')
+
+    def test_radiusaccounting_change(self):
+        options = dict(username='bobby', session_id='1')
+        obj = self._create_radius_accounting(**options)
```

Comment:
you are getting redirected in the test to the login form because you have not logged-in as an admin, check out how other tests do this in this or other modules, is quite common